zproject/settings.py

```diff
@@ -149,6 +149,8 @@ def get_secret(key):
                     'DBX_APNS_CERT_FILE': None,
                     'EXTRA_INSTALLED_APPS': [],
                     'DEFAULT_NEW_REALM_STREAMS': ["social", "general", "zulip"],
+                    # Used by the generate_realm_creation_link command. Value is in days.
+                    'REALM_CREATION_LINK_VALIDITY': 7,
```

Comment:
But I don't feel strongly on this one.  If you do stick with days, I'd call the variable REALM_CREATION_LINK_VALIDITY_DAYS to make the units obvious without having to look it up.